packages/headless/src/index.ts

```diff
@@ -550,6 +550,7 @@ export {buildResultTemplatesManager} from './features/result-templates/result-te
 export * from './features/analytics/search-analytics-actions-loader';
 export * from './features/analytics/click-analytics-actions-loader';
 export * from './features/analytics/generic-analytics-actions-loader';
+export * from './features/actions-history/ipx-actions-history-actions-loader';
```

Comment:
I feel like this goes here and not in the region grouped section, but I'm guessing more than anything

Reply:
The problem is the IPX naming: There's no "IPX" package in Headless like there is for commerce, case assist, insight panel. It's confusing.

I would export it as part of classic/generic analytics functionality.

ie: 
* Delete `features/actions-history/ipx-actions-history-actions-loader.ts`
* Delete `features/actions-history/ipx-actions-history-actions.ts`
* Add the action in `features/analytics/analytics-action.ts`
* Export it in `features/analytics/generic-analytics-action-loader.ts`